core/embed/rust/src/ui/model_mercury/bootloader/mod.rs

```diff
@@ -77,6 +97,70 @@ impl ModelMercuryFeatures {
             Self::fadein();
         }
     }
+
+    #[cfg(feature = "new_rendering")]
+    fn screen_progress(
+        text: &str,
+        progress: u16,
+        initialize: bool,
+        fg_color: Color,
+        bg_color: Color,
+        icon: Option<(Icon, Color)>,
+        center_text: Option<&str>,
+    ) {
+        if initialize {
+            Self::fadeout();
+        }
+        display::sync();
+
+        render_on_display(None, Some(bg_color), |target| {
+            shape::Text::new(PROGRESS_TEXT_ORIGIN, text)
+                .with_font(Font::NORMAL)
+                .with_fg(BLD_FG)
+                .render(target);
+
+            let loader_offset: i16 = 19;
+            let center_text_offset: i16 = 10;
+            let center = SCREEN.center() + Offset::y(-loader_offset);
```

Comment:
negative offset is wrong

Reply:
fixed 74dc53c1bfd6d1b3c6c8a95b947bec0a7d219645